test/fixtures/classic-to-nested/example-1/input.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
```

Comment:
I'm curious, can we get away with a single input that is shared between `classic-to-flat` and `classic-to-nested` then only duplicate that the outputs are different?

Reply:
That'd be nice. I can try, but think we can treat this as a low priority.

The input and output files _seemed_ to need to be placed in a particular manner due to the library `fixturify`. I had to create a subfolder in `classic-to-flat` and `classic-to-nested` (which I called `example-1`) and place the files in that subfolder.

Reply:
> That'd be nice. I can try, but think we can treat this as a low priority.

👍 seems good, would you mind making an issue in the repo for this future cleanup? Just hoping to make it easier for folks to pick it up...

Reply:
Yep, will do!